package.json

```diff
@@ -18,7 +18,7 @@
     "text-encoding": "^0.7.0"
   },
   "peerDependencies": {
-    "expo-linking": "*",
```

Comment:
Apologies for the late review @uffe 
I don't quite understand why the `*` on the left should not work sufficiently?
It should allow the installer to install whatever version they want alongside.

Reply:
I assumed the same, but an older version not compatible with expo sdk51 was pulled in.

Reply:
`*` should allow all versions, does not automatically pull in a specific version, what if you manually install a newer version?

Reply:
Using `npm install --save expo-linking@6.3.1`?
What does `npm ls expo-linking` show?